scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.sql._
 import scala.collection.JavaConverters._
 
 class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOModel] with H2OMOJOPrediction {
-
+  H2OMOJOCache.startCleanupThread()
```

Comment:
should be static (=part of object) no?

Reply:
I'm afraid that there isn't an equivalent to Java static block in Scala. A companion object is just an instance of a different JVM class.

Reply:
yes, sorry, i meant companion object.

Reply:
hmm, ran a test - my misunderstanding 🤦‍♂